posthog/api/dead_letter_queue.py

```diff
@@ -0,0 +1,91 @@
+from typing import Any, Dict, List, Union
+
+from rest_framework import viewsets
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from ee.clickhouse.client import sync_execute
+from posthog.permissions import IsStaffUser
+from posthog.version import VERSION
+
+
+class DeadLetterQueueViewSet(viewsets.ViewSet):
+    """
+    Show info about instance for this user
+    """
+
+    permission_classes = [IsStaffUser]
+
+    def list(self, request: Request) -> Response:
+
+        metrics: List[Dict[str, Union[str, bool, int, float, Dict[str, Any]]]] = []
+
+        metrics.append(
+            {"key": "dlq_size", "metric": "Total events in dead letter queue", "value": get_dead_letter_queue_size()}
+        )
+
+        metrics.append(
+            {
+                "key": "dlq_events_last_24h",
+                "metric": "Events sent to dead letter queue in the last 24h",
+                "value": get_dead_letter_queue_size(),
+            }
+        )
```

Comment:
not used and `get_dead_letter_queue_size` doesn't look correct either

Reply:
ah yup typo from ctrl c ctrl v